easybuild/easyconfigs/m/MATLAB-Engine/MATLAB-Engine-2024b-Python-3.11.5-24.2.2-GCCcore-13.2.0.eb

```diff
@@ -0,0 +1,43 @@
+# Updated to newer versions of MATLAB
+# Author: J. Saßmannshausen (Imperial College London/UK)
+
+easyblock = 'PythonBundle'
+
+name = 'MATLAB-Engine'
+_matlab_version = '2024b'
+_engine_version = '24.2.2'
+_python_version = '3.11.5'
```

Comment:
Why introduce this? We don't add python version as a suffix for anything else in GCCcore.

Reply:
It requires Python and as `MATLAB` is `SYSTEM`, I needed a way to get Python in it. So that was my attempt to get that somehow working. At least this way the user sees which Python is being used. 
And yes, I agree, that is not what we usually do. Do you have any better ideas?

Reply:
Just don't add the suffix.

Reply:
Done

Reply:
you missed removing the suffixes version from the pr

Reply:
My wrong. All done now. Sorry for that.